.gitignore

```diff
@@ -4,9 +4,12 @@ composer.lock
 
 # Cache
 /tests/cache/
-/demo/cache/
+/demo/src/cache/
```

Comment:
That was not what I meant :)
Could you change the cache path in the settings for the demo to point to `/demo/cache/`

Reply:
Ha ! I see. Ok let's try